Misc/NEWS.d/next/Security/2021-01-31-05-28-14.bpo-43075.DoAXqO.rst

```diff
@@ -0,0 +1 @@
+Fix Regular Expression Denial of Service (ReDoS) vulnerability in urllib.request.  The ReDoS-vulnerable regex has quadratic worst-case complexity and it allows cause a denial of service when identifying crafted invalid RFCs. This issue only affects consumers using urllib.request.AbstractBasicAuthHandler.
```

Comment:
```suggestion
Fix Regular Expression Denial of Service (ReDoS) vulnerability in :class:`urllib.request.AbstractBasicAuthHandler`.  The ReDoS-vulnerable regex has quadratic worst-case complexity and it allows cause a denial of service when identifying crafted invalid RFCs.
```

Please explain that the issue is on the client server, the attacker needs to control the HTTP server.